src/net.h

```diff
@@ -1024,6 +1024,9 @@ class CConnman
 
     void SetAsmap(std::vector<bool> asmap) { addrman.m_asmap = std::move(asmap); }
 
+    /** Return true if the peer has been connected for long enough to do inactivity checks. */
```

Comment:
can't tell if you're responding to each other or to the comment... but "connected for long enough" = connected for `m_peer_connect_timeout`, otherwise it's too soon to run inactivity checks

Reply:
are y'all talking about the comment being unclear or ?

Reply:
#21198